packages/chevrotain/src/scan/lexer.ts

```diff
@@ -219,7 +219,9 @@ export function analyzeTokenTypes(
       const longerAltType = clazz.LONGER_ALT
 
       if (longerAltType) {
-        const longerAltIdx = indexOf(onlyRelevantTypes, longerAltType)
+        const longerAltIdx = Array.isArray(longerAltType)
```

Comment:
- use the `isArray` function for the utility package to be consistent with the rest of the code base
  - https://github.com/Chevrotain/chevrotain/blob/239c59fd48ea650b21fb31b3f2492fe1aa865b44/packages/utils/src/api.ts#L254-L256

- This modification changes the return type of the scanner analysis function.
   This means the relevant interface must also be updated (`number` -> `number[]`)
   - https://github.com/Chevrotain/chevrotain/blob/239c59fd48ea650b21fb31b3f2492fe1aa865b44/packages/chevrotain/src/scan/lexer.ts#L47-L65